keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/ResourceFactory.kt

```diff
@@ -0,0 +1,28 @@
+package com.netflix.spinnaker.keel.sql
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.netflix.spinnaker.keel.api.Resource
+import com.netflix.spinnaker.keel.api.ResourceKind
+import com.netflix.spinnaker.keel.resources.ResourceSpecIdentifier
+import com.netflix.spinnaker.keel.resources.SpecMigrator
+import com.netflix.spinnaker.keel.resources.migrate
+
+internal class ResourceFactory(
```

Comment:
I like this approach. Can you add some comments here, like about when to use it?

Reply:
Done!